java/arcs/android/devtools/DevToolsStarter.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.devtools
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.core.content.ContextCompat.startForegroundService
+import arcs.core.util.TaggedLog
+import arcs.sdk.android.storage.service.StorageService
+
+/**
+ * The DevToolsStarter class can be used to start the [DevToolsService]. Requires the [Context]
+ * to start the [DevToolsService].
+ */
+class DevToolsStarter(private val context: Context) {
+
+  /**
+   * Start the [DevToolsService] with an optional custom [StorageService] class.
+   */
+  fun start(storageClass: Class<*> = StorageService::class.java) {
+    // Check if the Class received is a child of the StorageService.
+    if (!StorageService::class.java.isAssignableFrom(storageClass)) {
+      throw Error("StorageClass received is not a child of StorageService")
+    }
```

Comment:
It's not recommended to throw a generic error like this, instead it's preferred to use something which makes more sense in-context.  If you use `require`, it'll throw an `IllegalArgumentException` if the check fails, which is much more appropriate here.

```suggestion
    require(StorageService::class.java.isAssignableFrom(storageClass)) {
      "$storageClass is not a child of StorageService"
    }
```